### PR TITLE
Support filtering on epoch range on transfers

### PR DESCRIPTION
**Pull request by ValentinTrinque**

### CODE REVIEW

**Review by candida-d — DISMISSED** [review]

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 1]
```diff
@@ -1727,6 +1727,10 @@ message ListTransfersRequest {
   optional Pagination pagination = 3;
   // Restrict transfers to those related to reward transfers.
   optional bool is_reward = 4;
+  // Epoch to filter from (included). If omitted, `to epoch` must not be set. If both omitted, the most recent epoch's data is returned.
```
Comment (wwestgarth) [inline protos/sources/data-node/api/v2/trading_data.proto:1730]: Any reason why this doesn't work like any of the "range" filters in other API, and allow a `to` to be set without a `from` and you just get everything from `0` up to the `to`?

Comment (ValentinTrinque) [reply protos/sources/data-node/api/v2/trading_data.proto:1730]: We made this choice because we couldn't find any reason to restrict this.

**Review by gordsport — DISMISSED** [review]

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 2]
```diff
@@ -1730,8 +1730,12 @@ message ListTransfersRequest {
   TransferDirection direction = 2;
   // Pagination controls.
   optional Pagination pagination = 3;
-  // Restrict transfers to those related to reward transfers.
+  // Restrict transfers to athose related to reward transfers.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1733]:
```suggestion
  // Restrict transfers to those related to reward transfers.
```

Comment (candida-d) [reply protos/sources/data-node/api/v2/trading_data.proto:1733]: Tried to sneak this one by... ;)

**Review by guoguojin — DISMISSED** [review]

**Review by MadalinaRaicu — DISMISSED** [review]

**Review by candida-d — APPROVED** [review]

**Review by guoguojin — APPROVED** [review]

**Pull request merged**